DOCS/man/mpv.rst

```diff
@@ -711,22 +711,25 @@ or at runtime with the ``apply-profile <name>`` command.
         # a profile that can be enabled with --profile=big-cache
         [big-cache]
         cache=yes
-        demuxer-max-bytes=123400KiB
+        demuxer-max-bytes=512MiB
         demuxer-readahead-secs=20
 
-        [slow]
-        profile-desc="some profile name"
-        # reference a builtin profile
-        profile=high-quality
+        [network]
+        profile-desc="profile for content over network"
+        profile-cond=demuxer_via_network
```

Comment:
This can be confusing since `profile-cond` is not explained until the 2nd next section, but you could add it next to youtube since it's a useful example,

Reply:
just removed the `profile-cond` instead